site/_collections/events.js

```diff
@@ -15,86 +15,145 @@
  */
 
 const authorsData = require('../_data/authorsData.json');
-const PLACEHOLDER_IMG =
+const {i18n} = require('../_filters/i18n');
+const {Img} = require('../_shortcodes/Img');
+
+const AUTHOR_PLACEHOLDER =
   'image/tcFciHGuF3MxnTr1y5ue01OGLBn2/PFaMfvDZoPorronbpdU8.svg';
```

Comment:
We have defaults like these set site-wide in site.json: https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_data/site.json#L21
Please pull from there / add new if necessary.